### PR TITLE
Add django rest pandas

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by sannykr** [comment]
@conda-forge-admin : Could you please review and merge this pull request.

File: recipes/rest-pandas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
```
Comment (isuruf) [inline recipes/rest-pandas/meta.yaml]: Why this version requirement?

File: recipes/rest-pandas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - djangorestframework >=3.3.1
+    - pandas >=0.19.0
+    
+about:
+  home: https://github.com/wq/django-rest-pandas
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (isuruf) [inline recipes/rest-pandas/meta.yaml]: Can you open an issue or a PR upstream for packaging the LICENSE file?

Comment (sannykr) [reply recipes/rest-pandas/meta.yaml]: done

File: recipes/rest-pandas/meta.yaml (hunk 1 above)
Comment (sannykr) [reply recipes/rest-pandas/meta.yaml]: Thanks for pointing out.

**Comment by sannykr** [comment]
@isuruf : I have a question. In the setup.py file, I can see some test requirements. Do I need to add them to test: requires section?

**Pull request closed without merging**